Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -223,15 +278,20 @@ class SearchSettingsTableViewController: ThemedTableViewController, FeatureFlagg
             navigationController?.pushViewController(customSearchEngineForm, animated: true)
         case .privateSession:
             return nil
+        case .firefoxSuggestSettings:
+            guard indexPath.item == 2 else { return nil }
```

Comment:
```suggestion
            guard indexPath.item == ItemSuggestionLearn else { return nil }
```

(Since you already have the constant 😊)